### PR TITLE
471 configure swervechassis for practice bot

### PR DESCRIPTION
**Pull request by witcpalek**
Field Drive works now and removed a bunch of logging

### CODE REVIEW

File: vendordeps/photonlib.json [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
-    },
-    {
-      "groupId": "org.photonvision",
-      "artifactId": "photontargeting-java",
-      "version": "v2024.2.2"
-    }
-  ]
+    "fileName": "photonlib.json",
+    "name": "photonlib",
+    "version": "v2024.2.5",
```
Comment (nickfrisch) [inline vendordeps/photonlib.json:4]: Change to 2024.2.2

File: vendordeps/photonlib.json [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
+                "windowsx86-64",
+                "linuxathena",
+                "linuxx86-64",
+                "osxuniversal"
+            ]
+        },
+        {
+            "groupId": "org.photonvision",
+            "artifactId": "photontargeting-cpp",
+            "version": "v2024.2.5",
```
Comment (nickfrisch) [inline vendordeps/photonlib.json:32]: change o 2024.2.2

File: vendordeps/photonlib.json [hunk 3]
```diff
@@ -1,57 +1,57 @@
...
+                "linuxx86-64",
+                "osxuniversal"
+            ]
+        }
+    ],
+    "javaDependencies": [
+        {
+            "groupId": "org.photonvision",
+            "artifactId": "photonlib-java",
+            "version": "v2024.2.5"
```
Comment (nickfrisch) [inline vendordeps/photonlib.json:49]: change to 2024.2.2

File: vendordeps/photonlib.json [hunk 4]
```diff
@@ -1,57 +1,57 @@
...
+        },
+        {
+            "groupId": "org.photonvision",
+            "artifactId": "photontargeting-java",
+            "version": "v2024.2.5"
```
Comment (nickfrisch) [inline vendordeps/photonlib.json:54]: change to 2024.2.2

File: vendordeps/photonlib.json [hunk 5]
```diff
@@ -1,57 +1,57 @@
...
+        "https://maven.photonvision.org/repository/internal",
+        "https://maven.photonvision.org/repository/snapshots"
+    ],
+    "jsonUrl": "https://maven.photonvision.org/repository/internal/org/photonvision/photonlib-json/1.0/photonlib-json-1.0.json",
+    "jniDependencies": [],
+    "cppDependencies": [
+        {
+            "groupId": "org.photonvision",
+            "artifactId": "photonlib-cpp",
+            "version": "v2024.2.5",
```
Comment (nickfrisch) [inline vendordeps/photonlib.json:17]: change to 2024.2.2

**Review by nickfrisch — CHANGES_REQUESTED** [review]

File: vendordeps/photonlib.json (hunk 1 above)
Comment (witcpalek) [reply vendordeps/photonlib.json]: changed

File: vendordeps/photonlib.json (hunk 2 above)
Comment (witcpalek) [reply vendordeps/photonlib.json]: changed

File: vendordeps/photonlib.json (hunk 3 above)
Comment (witcpalek) [reply vendordeps/photonlib.json]: changed

File: vendordeps/photonlib.json (hunk 4 above)
Comment (witcpalek) [reply vendordeps/photonlib.json]: changed

File: vendordeps/photonlib.json (hunk 5 above)
Comment (witcpalek) [reply vendordeps/photonlib.json]: changed

**Pull request merged**